Module-11/Library/Documents/IDocument.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace Library.Documents;
```

Comment:
please update the way entities are organized:
move IDocument.cs to Library.Interfaces or just root Library namespace
move Bool, LocalizedBook, Magazine and Patent to Library.Models namespace